fast-vm

```diff
@@ -212,6 +212,13 @@ calculate_day_difference () {
 	echo "$((($end-$start)/86400))d"
 }
 
+use_sshkey_if_exists () {
+# If we our ssh key exists, set up to use it
+	if [ -f "$FASTVM_USER_CONF_DIR/id_fastvm" ]; then
+		FASTVM_USE_SSHKEY="-i $FASTVM_USER_CONF_DIR/id_fastvm -o BatchMode=yes"
```

Comment:
Why we add the `-o BatchMode=yes`? 
When the key is found, but not present on VM, this will prevent us from logging into machine using just password.

Reply:
Indeed you are correct.  It came from testing an earlier version and not needed.  Removed in the latest code, thanks.